axelar-chains-config/src/utils/sourcify.js

```diff
@@ -0,0 +1,140 @@
+'use strict';
+
+const axios = require('axios');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+async function findValue(obj, contractName, result = []) {
+    if (obj && typeof obj === 'object') {
+        for (const key in obj) {
+            if (key.includes(`/${contractName}.sol`) || key === contractName) {
+                result.push(obj[key].metadata ? obj[key].metadata : obj[key].content);
+            }
+
+            await findValue(obj[key], contractName, result);
+        }
+    }
+}
+
+async function readAndParseFile(filePath, targetKey) {
+    try {
```

Comment:
avoid adding try/catch in helper methods. In most cases, the caller should decide how to handle an error, or ensure the method shouldn't error, so we don't continue on error